package.json

```diff
@@ -53,8 +58,8 @@
     "mocha": "^5.0.4",
     "mocha-chrome": "^1.0.3",
     "moment": "~2.21.0",
-    "ms-rest-azure-js": "Azure/ms-rest-azure-js#GitHubReference",
-    "ms-rest-js": "Azure/ms-rest-js",
+    "ms-rest-azure-js": "RikkiGibson/ms-rest-azure-js#HttpClient",
+    "ms-rest-js": "RikkiGibson/ms-rest-js#HttpClient",
```

Comment:
Until @amarzavery approves publishing a new version of ms-rest-js, we really should leave these pointing to ms-rest-js and ms-rest-azure-js#GitHubReference.

Reply:
This is to make CI pass so we can confirm these changes are ok before merging/publishing on the ms-rest-js side.